lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -670,6 +673,16 @@ void FIRRTLModuleLowering::runOnOperation() {
               state.recordModuleMapping(&op, loweredMod);
               return success();
             })
+            .Case<FormalOp>([&](auto oldFormalOp) {
+              auto builder = OpBuilder::atBlockEnd(topLevelModule);
+              auto newFormalOp = builder.create<verif::FormalOp>(
+                  oldFormalOp.getLoc(), oldFormalOp.getNameAttr());
+              newFormalOp->setDiscardableAttrs(oldFormalOp.getParametersAttr());
```

Comment:
Maybe we want some ODS verified dictionary attribute for the `verif.formal` op instead of using a discardable attribute?

Reply:
We we should definitely do that. I'll add that in a follow-up PR.